File: client/src/app/shared/models/service.ts
import { Photo } from "./photo"

export interface Service {
  id: number
  name: string
  description: string
  servicePhotos: ServicePhoto[]
  serviceCategory: ServiceCategory
  userService: UserService
}

interface ServicePhoto {
  photoId: number
  photo: Photo
}

interface ServiceCategory {
  serviceId: number
  categoryId: number
  category: Category
}

export interface Category {
  id: number
  name: string
}

interface UserService {
  userId: number
  user: User
  serviceId: number
}

interface User {
  id: number
  phoneNumber: string
  email: string
  firstName: string
  lastName: string
  dateOfBirth: string
  facebook: string
  instagram: string
}

File: Core/Entities/Service.cs
namespace Core.Entities
{
    public class Service : BaseEntity
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public List<ServicePhoto> ServicePhotos { get; set; }
        public ServiceCategory ServiceCategory { get; set; }
        public UserService UserService { get; set; }
    }
}